projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh

```diff
@@ -134,29 +135,37 @@ env_and_envsubst '$EKSD_NAME' \
 
 # This is the IP address that Packer will create the server on to host the local
 # directory containing the kickstart config
-if [ "$IMAGE_FORMAT" = "ova" ] && [ "$IMAGE_OS" = "redhat" ]; then
+if [ "$IMAGE_FORMAT" = "ova" ]; then
+    echo "Finding correct interface for packer temporary http server"
     ACTIVE_INTERFACE=""
     if [ "$(uname -s)" = "Linux" ]; then
         INTERFACES=($(ls /sys/class/net))
         for interface in "${INTERFACES[@]}"; do
             if [ "$interface" = "eth0" ] || [ "$interface" = "en0" ] || [ "$interface" = "eno1" ]; then
                 ACTIVE_INTERFACE=$interface
+                echo "Found interface: $interface"
                 break
             fi
         done
+        HTTP_SERVER_IP=$(ip a l $ACTIVE_INTERFACE | awk '/inet / {print $2}' | cut -d/ -f1)
     elif [ "$(uname -s)" = "Darwin" ]; then
         ACTIVE_INTERFACE="en0"
+        HTTP_SERVER_IP=$(ifconfig $ACTIVE_INTERFACE | awk '/inet / {print $2}' | cut -d/ -f1)
     fi
+    
     if [ -z $ACTIVE_INTERFACE ]; then
         echo "ACTIVE_INTERFACE cannot be an empty string. Please check your network configuration
         and set an appropriate value for ACTIVE_INTERFACE"
         exit 1
     fi
-    export PACKER_HTTP_SERVER_IP=$(ip a l $ACTIVE_INTERFACE | awk '/inet / {print $2}' | cut -d/ -f1)
-    rhel_ova_config_file="${MAKE_ROOT}/${IMAGE_BUILDER_DIR}/packer/ova/rhel-8.json"
-    echo "Setting boot_media_path for rhel config to $PACKER_HTTP_SERVER_IP"
-    cat <<< $(jq --arg httpendpoint "http://$PACKER_HTTP_SERVER_IP:{{ .HTTPPort }}" \
-             '.boot_media_path=$httpendpoint' $rhel_ova_config_file) > $rhel_ova_config_file
+
+    PACKER_HTTP_SERVER_IP=${PACKER_HTTP_SERVER_IP:-$HTTP_SERVER_IP}
+    if [ -z $PACKER_HTTP_SERVER_IP ]; then
+        echo "PACKER_HTTP_SERVER_IP cannot be automatically determined.  Please export PACKER_HTTP_SERVER_IP=<Current Host's IP>"
+        exit 1
+    fi
+
+    find ${MAKE_ROOT}/${IMAGE_BUILDER_DIR}/packer/ova -type f -name '*.json' | xargs sed -i "s/{{ .HTTPIP }}/$PACKER_HTTP_SERVER_IP/g"
```

Comment:
Need this check to use the proper sed executable for Darwin
```suggestion
    SED=sed
    if [ "$(uname -s)" = "Darwin" ]; then
      SED=gsed
    fi
    find ${MAKE_ROOT}/${IMAGE_BUILDER_DIR}/packer/ova -type f -name '*.json' | xargs $SED -i "s/{{ .HTTPIP }}/$PACKER_HTTP_SERVER_IP/g"
```